Exercise1.js

```diff
@@ -0,0 +1,12 @@
+function nestingLevel(arr) {
+    if (!Array.isArray(arr)) return 0;
+    let maxLevel = 0;
+    for (let item of arr){
+        maxLevel = Math.max(maxLevel, nestingLevel(item));
+    }
+    return maxLevel + 1;
+}
```

Comment:
It's a good readability practice to leave empty line before `return` statement and before and after a loop or a conditional statement!

```suggestion
function nestingLevel(arr) {
    if (!Array.isArray(arr)) return 0;
    let maxLevel = 0;
    
    for (let item of arr){
        maxLevel = Math.max(maxLevel, nestingLevel(item));
    }
    
    return maxLevel + 1;
}
```